packages/core/admin/admin/src/pages/SettingsPage/pages/Webhooks/EditView/components/Events/index.js

```diff
@@ -0,0 +1,288 @@
+import * as React from 'react';
+import PropTypes from 'prop-types';
+
+import { useFormikContext } from 'formik';
+import { useIntl } from 'react-intl';
+import styled from 'styled-components';
+import { FieldLabel, Flex, Typography, BaseCheckbox, Checkbox } from '@strapi/design-system';
+
+import { useModels } from '../../../../../../../hooks';
+
+export const formatValue = (value) =>
+  value.reduce((acc, curr) => {
+    const key = curr.split('.')[0];
+
+    if (!acc[key]) {
+      acc[key] = [];
+    }
+    acc[key].push(curr);
+
+    return acc;
+  }, {});
+
+const StyledTable = styled.table`
```

Comment:
Why do we need a custom table here?

Reply:
Do you think we should be using the Table or RawTable from the DS?

Reply:
If it is possible - yes definitely. In case something is missing we could check if we can add it to the DS. I'd say in the CMS consistency wins in a case like this.